src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text.txt

```diff
@@ -27,39 +26,31 @@
 #Error
 
 >> Text(0.5, ".##")
-/*InterpreterRunner*/ #Skip
 ".5"
 
 >> Text(1.5, "000")
-/*InterpreterRunner*/ #Skip
 "002"
 
 >> Text(123456789, "#,#")
-/*InterpreterRunner*/ #Skip
 "123,456,789"
 
 >> Text(123456789, "#,###")
-/*InterpreterRunner*/ #Skip
 "123,456,789"
 
 >> Text(123456789, "0,0.0")
-/*InterpreterRunner*/ #Skip
 "123,456,789.0"
 
 >> Text(123456789, "0,00.0")
-/*InterpreterRunner*/ #Skip
 "123,456,789.0"
 
 >> Text(123456789, "0,")
-/*InterpreterRunner*/ #Skip
-"123456789,"
+"123457"
```

Comment:
Has this been tested in the TypeScript implementation?

Reply:
I'll have to get with you on how to test against the TypeScript implementation as I'm not sure how to do that. I did check against Excel and this is what it returns when provided the same input.

Reply:
I wonder why Excel does it this way. You can also use the prod PowerFx client to test by hand.

Reply:
I just checked in Canvas Apps and the original test was correct. We can ask Greg why this is the case.

Reply:
I've added an expected result for the InterpreterRunner since this is the value we expect based on Excel. If TypeScript is different then they need to work out why, but the base expected result is now the original "123456789," value.